constants.js

```diff
@@ -1,51 +1,54 @@
 const folderMapping = {
-  android: 'Android',
-  flutter: 'Flutter',
-  ios: 'iOS',
-  react: 'React',
-  reactnative: 'React Native',
-  angular: 'Angular',
-  api: 'API',
-};
+  android: "Android",
+  flutter: "Flutter",
+  ios: "iOS",
+  react: "React",
+  reactnative: "React Native",
+  angular: "Angular",
+  api: "API",
+}
 
 const platformMapping = {
-  android: 'android',
-  flutter: 'flutter-dart',
-  ios: 'ios-swift',
-  react: 'react',
-  reactnative: 'react-native',
-  angular: 'angular',
-  api: 'api',
-};
+  android: "android",
+  flutter: "flutter-dart",
+  ios: "ios-swift",
+  react: "react",
+  reactnative: "react-native",
+  angular: "angular",
+  api: "api",
+}
 
 const languageMapping = {
-  android: 'kotlin',
-  flutter: 'dart',
-  ios: 'swift',
-  react: 'javascript',
-  reactnative: 'javascript',
-  angular: 'javascript',
-  api: 'javascript',
-};
+  android: "kotlin",
+  flutter: "dart",
+  ios: "swift",
+  react: "javascript",
+  reactnative: "javascript",
+  angular: "javascript",
+  api: "javascript",
+}
 
-const PRODUCT = process.env.PRODUCT;
+const PRODUCT = process.env.PRODUCT
 let DOCUSAURUS_INDEX =
-  process.env.DEPLOYMENT_ENV === 'production'
-    ? `DOCUSAURUS${PRODUCT === 'video' ? '_VIDEO' : ''}`
-    : `DOCUSAURUS${PRODUCT === 'video' ? '_VIDEO' : ''}_STG`;
-const CMS_INDEX = 'DOCS';
+  process.env.DEPLOYMENT_ENV === "production"
+    ? `DOCUSAURUS${PRODUCT === "video" ? "_VIDEO" : ""}`
+    : `DOCUSAURUS${PRODUCT === "video" ? "_VIDEO" : ""}_STG`
+const CMS_INDEX = "DOCS"
 
-const IGNORED_DIRECTORIES = ['common-content'];
+const IGNORED_DIRECTORIES = ["common-content"]
 
 const SDK_ORDER = [
-  'react',
-  'ios',
-  'android',
-  'reactnative',
-  'flutter',
-  'angular',
-  'api',
-];
+  "react",
+  "ios",
+  "android",
+  "reactnative",
+  "flutter",
+  "angular",
+  "api",
+]
+
+const ignoredVideoSDKsStaging = ["angular"]
```

Comment:
@DaemonLoki is this the part that you've introduced to filter docs for staging and prod?